src/dattobd.c

```diff
@@ -90,17 +90,23 @@ static loff_t noop_llseek(struct file *file, loff_t offset, int origin){
 
 
 #ifndef HAVE_STRUCT_PATH
```

Comment:
why is this still `HAVE_STRUCT_PATH`? should this be changed to `HAVE_PATH_PUT` as well?